core/authority-discovery/src/lib.rs

```diff
@@ -62,8 +62,14 @@ use std::time::{Duration, Instant};
 
 mod error;
 /// Dht payload schemas generated from Protobuf definitions via Prost crate in build.rs.
+#[cfg(not(target_os = "windows"))]
 mod schema {
-	include!(concat!(env!("OUT_DIR"), "/authority_discovery.rs"));
+    include!(concat!(env!("OUT_DIR"), "/authority_discovery.rs"));
+}
+
+#[cfg(target_os = "windows")]
+mod schema {
+    include!(concat!(env!("OUT_DIR"), r"\authority_discovery.rs"));
```

Comment:
I am not that familiar with windows, but AFAIR it supports `/`-style paths, isn't it?

Reply:
Yeah, Windows supports slash separators as well.

Reply:
so... this doesn't actually fix anything?

Reply:
it shouldn't according to my mental model. However, I am not 100% sure since we don't do windows builds AFAIK.